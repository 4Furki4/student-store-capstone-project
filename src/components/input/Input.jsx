import React from "react";

export default function Input({ placeholder, type }) {
    return (
        <>
            <input
                className=' my-3.5 block w-full rounded-lg border
                 border-gray-300 bg-gray-50 p-2.5 text-center text-xl
                  text-hadrisblue placeholder-hadrisblue focus:border-blue-500
                   focus:ring-purple dark:border-gray-600 dark:bg-gray-700 dark:text-white
                    dark:focus:border-blue-500 dark:focus:ring-blue-500'
                id={type}
                type={type}
                placeholder={placeholder}
                required
            />
        </>
    );
}
