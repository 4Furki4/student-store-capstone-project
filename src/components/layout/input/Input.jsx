import React from "react";

export default function Input({ placeholder, type }) {
    return (
        <>
            <input
                className=' text-md my-3.5 block w-full rounded-lg
                 border border-gray-300 bg-gray-50 p-2.5 text-center
                  text-hadrisblue placeholder-hadrisblue placeholder:text-sm focus:border-blue-500
                   focus:ring-purple'
                id={type}
                type={type}
                placeholder={placeholder}
                required
            />
        </>
    );
}
