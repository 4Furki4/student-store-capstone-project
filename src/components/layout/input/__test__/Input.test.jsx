import renderer from "react-test-renderer";

import Input from "../index";

it("renders correctly", () => {
    const tree = renderer.create(<Input />).toJSON();
    expect(tree).toMatchSnapshot();
});
