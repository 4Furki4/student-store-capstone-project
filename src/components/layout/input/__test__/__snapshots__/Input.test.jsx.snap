// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<input
  className=" my-3.5 block w-full rounded-lg border border-gray-300 bg-gray-50 p-2.5 text-center text-xl text-hadrisblue placeholder-hadrisblue focus:border-blue-500 focus:ring-purple"
  required={true}
/>
`;
